### DISCUSSION TITLE
How to add a microphone?

### DISCUSSION THREAD

**Discussion by Ricardo-Piccolo** [jspsych/jsPsych#3077] [Q&A] [answered]
Hello,

I have to build a verbal fluency task in which the participants have to say as many elements they can within a theme (e.g.: FRUITS: apple, banana, grapes...).

For that, I would really appreciate if I could add a microphone plugin/component to my experiment. The objective is that the microphone will be active during the whole experiment. How can I do that?

I'm using JsPsych version 6.1.0.

**Comment by jodeleeuw** [comment, marked as answer]
Hi @Ricardo-Piccolo,

There's a [plugin that can record audio](https://www.jspsych.org/7.3/plugins/html-audio-response/), but it is only compatible with jsPsych 7.

For making it active during the entire experiment, you'd need to use an [extension](https://www.jspsych.org/7.3/overview/extensions/). We have an extension to record video but not audio. It probably would be relatively few edits to turn the video recorder into an audio recorder.
